### PR TITLE
[Issue #887] Snippet metadata export cron job.

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by glogiotatidis** [comment]
@pmac note that i'm using Google's S3 compatibility layer. What we want to do is simple and does not require any google specific functionality. We also avoid adding more python dependencies this way. Thoughts?

**Comment by pmac** [comment]
I like it 👍 I assume there's a reason we can't just use S3? Either way, that seems fine.

**Comment by glogiotatidis** [comment]
> I assume there's a reason we can't just use S3?

I'm in discussions with the Data team to use S3 if possible

**Comment by glogiotatidis** [comment]
Waiting for final decisions on where we'll store the metadata file.

**Comment by glogiotatidis** [comment]
@pmac we changed that to upload to S3 directly. 

I plan to re-use the existing buckets and configure a random url like

   `/metadata/wu1me1no4de8ri1ca3cy/snippets_metadata_20190402.csv`

to share with the data egnineers.

can you please take a second look?

File: docs/architecture/decisions/0002-export-asrsnippet-metadata-for-snippet-metrics-processing.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+Accepted
+
+## Context
+
+Data Engineers are building performance dashboards for Snippets using the
+metrics we collect from Firefox Telemetry. Telemetry pings include only basic
+information about the Snippet, like Snippet ID.
+
+For better to understand ad more complete dashboards, we want to enhance the
```
Comment (metadave) [inline docs/architecture/decisions/0002-export-asrsnippet-metadata-for-snippet-metrics-processing.md]: can you fix this sentence?

        For better to understand ad more complete dashboards

Comment (glogiotatidis) [reply docs/architecture/decisions/0002-export-asrsnippet-metadata-for-snippet-metrics-processing.md]: done, thanks!

File: snippets/base/storage.py [hunk 2]
```diff
@@ -10,8 +10,17 @@
 
 
 class OverwriteStorage(FileSystemStorage):
+    """
+    Comes from http://www.djangosnippets.org/snippets/976/
+    (even if it already exists in S3Storage for ages)
```
Comment (pmac) [inline snippets/base/storage.py]: I'm confused about the mention of S3Storage here. This `OverwriteStorage` class doesn't seem to have anything to do with the S3 storage stuff.

Comment (glogiotatidis) [reply snippets/base/storage.py]: Fair comment. Not sure why this is there. I just refactored / copy-pasted from `snippets.storage` to `snippets.base.storage`. Will remove this comment make things clear.

**Comment by glogiotatidis** [comment]
Thanks @pmac!

**Pull request merged**